packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/di/{{project_name}}_with_dependencies.dart

```diff
@@ -242,7 +242,8 @@ class _{{project_name.pascalCase()}}WithDependenciesState extends State<{{projec
           create: (context) => UserAccountService(
             context.read(),
             context.read(),
-            context.read(),
+            context.read(),{{#enable_social_login}}
+            null,{{/enable_social_login}}
```

Comment:
Why we need to pass null ?

Reply:
We have positional nullable parameter. I kept them positional, to not refactor the old code and to be consistent ( we have named parameters in only one place). Will fix this.